README.rst

```diff
@@ -22,30 +27,41 @@ For now, this repo is not intergrated with devstack. You'll be running the app l
 
       mkdir -p ~/workspace/
       cd ~/workspace/
-      git clone https://github.com/edx/devstack.git
+      git clone https://github.com/edx/frontend-app-communications.git
 
 2. Install frontend dependencies
 
    .. code-block::
 
       npm i
 
-3. start the devserver. The app will be running at `localhost:8585`, or whatever port you change it too.
+3. start the devserver. The app will be running at ``localhost:1984``, or whatever port you change it too.
 
    .. code-block::
 
-      npm i
+      npm start
 
 Environment Variables/Setup Notes
 ---------------------------------
 
-TBD
+if you wish to add new environment varibles for local testing, they should be listed in 2 places:
+
+1. In ``.env.development``
+2. Added to the ``mergeConfig`` found in ``src/index.jsx``
+
+.. code-block::
+
+   initialize({
+      config: () => {
+       mergeConfig({
+       EXAMPLE_VALUE: true,
+       }, 'CommuncationsAppConfig');
```

Comment:
Should we add this to the cookie cutter app Readme? It seems like good advice for any MFE that is created.

Reply:
This information is listed else-where in the documentation for the cookie cutter. I just put it here for more vision into something we do fairly often.

Reply:
This information is listed else-where in the documentation for the cookie cutter. I just put it here for more vision into something we do fairly often.